src/Mongolid/ActiveRecord.php

```diff
@@ -113,8 +119,13 @@ public static function where(array $query = [])
      */
     public static function all()
     {
-        return Ioc::make(get_called_class())
-            ->getDataMapper()->all();
+        $instance = Ioc::make(get_called_class());
```

Comment:
we could extract to a `protected function getInstance()` to avoid code duplication